synapse/storage/databases/main/event_push_actions.py

```diff
@@ -516,6 +516,26 @@ async def get_unread_push_actions_for_user_in_range_for_http(
             ),
         )
 
+        def get_push_actions(
```

Comment:
just to be customary, this name should include the `_txn` suffix because it takes a Transaction.

Reply:
https://github.com/matrix-org/synapse/pull/13597/commits/55f587c6be3ad4acdb96c1b5746522830fea27b2